app/components/calls_to_action/lifeasateacher_block_component.rb

```diff
@@ -0,0 +1,28 @@
+module CallsToAction
+  class LifeasateacherBlockComponent < ViewComponent::Base
```

Comment:
might this class be better named `LifeAsATeacherBlockComponent` and the corresponding file named `app/components/calls_to_action/life_as_a_teacher_block_component.rb` ?

Reply:
..and in this case, also rename the html template to `app/components/calls_to_action/life_as_a_teacher_block_component.html.erb`